### ISSUE TITLE
Author versus Writer Modules for title

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/wai-curricula#471]
The term "author" in WCAG means anybody involved in creating websites, including developers, designers, and writers. Curricula have already

* [Foundation](https://www.w3.org/WAI/curricula/foundation-modules/)
* [Developer](https://www.w3.org/WAI/curricula/developer-modules/)
* [Designer](https://www.w3.org/WAI/curricula/designer-modules/)

The next part will focus more on the writing part.

Proposal is to:

* use "writer" in title
* relate the knowledge about writing to the ability of authoring tools to produce accessible content

**Comment by shawna-slh** [comment]
Some more context:
* "author" is 41 times in WCAG 2.1 and often is addressing the developer role &mdash; e.g., in _"This success criterion is primarily for Web authors who develop or script their own user interface components."_, the word "authors" is developers/coders.
* We currently have **[Tips](https://www.w3.org/WAI/tips/) for Writing**, Tips for Designing, Tips for Developing
* To get an idea of potential curricula content, see [Writer Modules early draft](https://github.com/w3c/wai-curricula/wiki/writer-modules-outline)
* Remember the [primary audience is people who teach courses](https://www.w3.org/WAI/curricula/#using-the-curricula).

**Comment by shawna-slh** [comment]
> relate the knowledge about writing to the ability of authoring tools to produce accessible content

I wonder if this is scope creep that we don't want to slide into?

**Comment by eoncins** [comment]
If the scope is to "relate the knowledge about writing to the ability of authoring tools to produce accessible content" it is very narrow.

Content authors as writers, and actually also editors should be able to select and assess at some level the accessibility of the authoring tool to create accessible content. Otherwise, it seems that accessibility is exclusively technology-driven and determined mainly by developers/designers. It is not only that an authoring tool allows you to add alt-text, but by whom and how the alt-text is provided.  

Under primary audience for "faculty lecturer" no content authors are mentioned:
faculty lecturer — selects topics from the foundation, developer, and designer modules to teach accessibility to computer science students. Is there any specific reason for that?

**Comment by daniel-montalvo** [comment]
Hey @eoncins 

> If the scope is to "relate the knowledge about writing to the ability of authoring tools to produce accessible content" it is very narrow.
> 
> Content authors as writers, and actually also editors should be able to select and assess at some level the accessibility of the authoring tool to create accessible content. Otherwise, it seems that accessibility is exclusively technology-driven and determined mainly by developers/designers. It is not only that an authoring tool allows you to add alt-text, but by whom and how the alt-text is provided.

We currently have: "employ authoring tools that produce accessible markup for ..." and "employ authoring tools that support the inclusion of ..."

However, assessing the accessibility of the authoring tool would require quite a significant testing effort from the content author's perspective. I am not sure if it is realistic to think of the content author as someone who will be testing the accessibility of what they write. We have not included these testing knowledge in the designers and developers part. 

My thought for this is requiring the instructors to have knowledge of authoring tools accessibility in the competencies section.

> Under primary audience for "faculty lecturer" no content authors are mentioned: faculty lecturer — selects topics from the foundation, developer, and designer modules to teach accessibility to computer science students. Is there any specific reason for that?

We were just not sure how much of authoring content is taught in computer science. Happy to revisit this for future updates.

**Comment by eoncins** [comment]
Hey @daniel-montalvo thanks for the message and the explanation.

> We currently have: "employ authoring tools that produce accessible markup for ..." and "employ authoring tools that support the inclusion of ..."
> 

OK!

> However, assessing the accessibility of the authoring tool would require quite a significant testing effort from the content author's perspective. I am not sure if it is realistic to think of the content author as someone who will be testing the accessibility of what they write. We have not included these testing knowledge in the designers and developers part.
> 

> My thought for this is requiring the instructors to have knowledge of authoring tools accessibility in the competencies section.
> 
OK. I do not refer to auditing the accessibility level of the authoring tool but knowing the accessibility features of the given tool. 

> We were just not sure how much of authoring content is taught in computer science. Happy to revisit this for future updates.

I understand then that this particular curricula on author/writer only applies to "computer science" students and other areas such as Linguistics or Humanities which are very much in need for this training are not considered.

**Comment by daniel-montalvo** [comment]
I think the audiences you mention are under the umbrella of content author and we should cover them. But I think we should focus more on our primary audience: instructors who are going to create courses for teaching content authors, irrespective of their field of study.

I will close this as it was originally about the overall naming for the resource and I invite you to keep the discussion going in #487
